src/pages/api/post-update-profil.ts

```diff
@@ -0,0 +1,44 @@
+// pages/api/createService.ts
+import { NextApiRequest, NextApiResponse } from 'next';
+import { Contract, ethers, Wallet } from 'ethers';
+import { config } from '../../config';
+import TalentLayerID from '../../contracts/ABI/TalentLayerID.json';
+import { getUserByAddress } from '../../queries/users';
+
```

Comment:
double check the naming of all files here, following these rules: 
- remove globally the post- prefix
- profil => profile (use exactly same words than the delegate function and contract function)
- here the function is call delegateUpdateProfileData so it must be update-profile-data.ts